docs/readmes/orc8r/deploy_using_juju.md

```diff
@@ -20,15 +21,15 @@ the project's [homepage](https://github.com/canonical/charmed-magma).
 
 ## Set up your management environment
 
-From a Ubuntu 20.04 machine, install the following tools:
+From an Ubuntu 20.04 machine, install the following tools:
```

Comment:
```suggestion
From a Ubuntu 20.04 machine, install the following tools:
```
The first version was ok

Reply:
My IntelliJ made me do it :D
Changed back.